src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlBulkCopy.cs

```diff
@@ -137,15 +137,28 @@ private enum ValueSourceType
         }
 
         // Enum for specifying SqlDataReader.Get method used
-        private enum ValueMethod : byte
```

Comment:
@Wraith2 do you see any issues w/ dropping `byte` to support the options listed below?

Reply:
additionally, should this be a `Flags` enum w/ bit-wise values?

Reply:
`int` is probably better for alignment anyway, if you use byte the processor is likely to sign expand it into an int as the native data size to work with so I say go for it. And yes, specify flags attribute if that's how it will be used.

Reply:
changed to flags

Reply:
By attention use of this enumeration in the `switch-case` statement, using `Flags` attribute potentially will create some bugs.
I propose to roll back these changes.

Reply:
flags no more!